readme.md

```diff
@@ -144,6 +144,7 @@ Thanks for contributing! 🦋🙌
 - [](# "repo-wide-file-finder") Enables the the File Finder keyboard shortcut (<kbd>t</kbd>) on Issues and Pull Request pages as well.
 - [](# "file-finder-buffer") [Lets you start typing your search immediately after invoking the File Finder (<kbd>t</kbd>), instead of having you wait for it to load first.](https://user-images.githubusercontent.com/1402241/75542106-1c811700-5a5a-11ea-8aa5-bea0472c59e2.gif)
 - [](# "preserve-file-finder-term") Preserves the search terms when navigating back and forth between the File Finder and the files.
+- [](# "clone-brach") [Clone a branch from the branches list](https://user-images.githubusercontent.com/16872793/76715710-e0111480-6703-11ea-8b47-6c428e83a4e3.png)
```

Comment:
1. Try zooming in first so screenshot will have a higher quality
2. Try including a bit more context so people will know what they're looking at